mods/ra/rules/vehicles.yaml

```diff
@@ -467,6 +467,42 @@ MNLY.AT:
 	RenderSprites:
 		Image: MNLY
 
+MNLY.ATB:
```

Comment:
nope that was a test case I was using to test the MinefieldDepth, Ill remove it.